frontend/src/metabase/lib/moderation.js

```diff
@@ -1,7 +1,47 @@
+import { t } from "ttag";
+
+export const MODERATION_ACTIONS = {
+  verification: {
+    name: "verification",
+    icon: "verified",
+    color: "brand",
+  },
+  flag: {
+    name: "flag",
+    icon: "warning_colorized",
+    color: "accent5",
+  },
+  question: {
+    name: "question",
+    icon: "clarification",
+    color: "accent2",
+  },
+};
+
+export const MODERATION_TEXT = {
```

Comment:
I liked the `lib/constants.js` file @alxnddr added and was wondering what you guys thought about putting more text constants into `metbase/lib/*`?

I was really burned by having to tweak dashboard parameters labeling several times in my last project because it'd result in a million cypress tests breaking and it wasn't always easy to search for all the strings. This way cypress can theoretically depend on this object as well, if possible (not sure if it can import this right now).

I don't feel very strongly about this or the structure I put it in. I just did it this way because `user/moderator` and moderation issue type will be frequent branching points for this UI code.

Reply:
Agree. It's definitely a good call to extract values like that. The key thing, I guess, is to find out the best place to keep them.

`metabase/lib/constants` looks like a good place for global application-wide values. Perhaps I miss the scope of the moderation feature, but now it looks pretty compact and well-isolated, so I'm not sure if it's a good place for its constants.

Maybe it would make sense for the plugin to keep its constants? Like `enterprise/frontend/src/metabase-enterprise/moderation/constants`? Plugins already contain their components, so why not keep constants in the same place? Especially if the plugin components use the constants already (`ModerationIssueActionMenu`)

Reply:
It looks like `enterprise/frontend/src/metabase-enterprise/moderation/constants` is a good place for that if these constants won't be used in other parts of the application. I'd keep `lib/constants.js` for application-wide things